go/vt/vtgate/planbuilder/migration.go

```diff
@@ -40,7 +41,7 @@ func validateThrottleParams(alterMigrationType sqlparser.AlterMigrationType, exp
 		// Unthrottling is like throttling with duration=0
 		duration = 0
 	default:
-		duration = time.Hour * 24 * 365 * 100
+		duration = throttle.DefaultAppThrottleDuration
```

Comment:
Unrelated to the PR, but I just realized this wasn't using the default expiration, and so was not inline with all other `throttle` commands.